database/rrdset.c

```diff
@@ -1789,7 +1789,7 @@ void rrdset_done(RRDSET *st) {
 after_second_database_work:
     st->last_collected_total  = st->collected_total;
 
```

Comment:
Hello @kklionz ,

You made another [PR](https://github.com/netdata/netdata/pull/12969/) to fix this, an I already reviewed it. My suggestion is to keep this change only in the new PR, or we will need to merge it, and after you rebase this PR we will have conditions to merge this.

Reply:
Hello @thiagoftsm ,
      Do you mean that modify https://github.com/netdata/netdata/pull/12969 to add https://github.com/netdata/netdata/pull/12949 related change and close https://github.com/netdata/netdata/pull/12949?

Reply:
Hello,

I am saying that you have the same fix in two different PRs, in my understanding it would be good to keep the #12969 and here you could address only the loop.

Best regards!

Reply:
Hello @thiagoftsm ,
    I will close  https://github.com/netdata/netdata/pull/12949